ote/src/cljs/ote/views/admin.cljs

```diff
@@ -78,60 +77,94 @@
+                           :display-select-all false}
+          [ui/table-row
+           [ui/table-header-column
+            {:class "table-header-wrap" :style {:width "7%"}}
+            "Id"]
+           [ui/table-header-column
+            {:class "table-header-wrap" :style {:width "15%"}}
+            "Y-tunnus"]
+           [ui/table-header-column
+            {:class "table-header-wrap" :style {:width "25%"}}
+            "Nimi"]
+           [ui/table-header-column
+            {:class "table-header-wrap" :style {:width "10%"}}
+            "GSM"]
+           [ui/table-header-column
+            {:class "table-header-wrap" :style {:width "10%"}}
+            "Puhelin"]
+           [ui/table-header-column
+            {:class "table-header-wrap" :style {:width "18%"}}
+            "Sähköposti"]
+           [ui/table-header-column
+            {:class "table-header-wrap" :style {:width "20%"}}
+            "Toiminnot"]]]
+         [ui/table-body {:display-row-checkbox false}
+          (doall
+            (for [{::t-operator/keys [id name gsm phone business-id ckan-group-id email] :as result} results]
+              ^{:key (::t-operator/id result)}
+              [ui/table-row {:selectable false}
+               [ui/table-row-column
+                (merge (stylefy/use-style style-base/table-col-style-wrap)
+                       {:width "7%"})
+                id]
+               [ui/table-row-column
+                (merge (stylefy/use-style style-base/table-col-style-wrap)
+                       {:width "15%"})
+                business-id]
+               [ui/table-row-column
+                (merge (stylefy/use-style style-base/table-col-style-wrap)
+                       {:width "25%"})
+                [:a {:href (str "#/transport-operator/" id)
+                     :on-click #(do
+                                  (.preventDefault %)
+                                  (e! (admin-controller/->EditTransportOperator business-id)))} name]]
+               [ui/table-row-column
+                (merge (stylefy/use-style style-base/table-col-style-wrap)
+                       {:width "10%"})
+                gsm]
+               [ui/table-row-column
+                (merge (stylefy/use-style style-base/table-col-style-wrap)
+                       {:width "10%"})
+                phone]
+               [ui/table-row-column
+                (merge (stylefy/use-style style-base/table-col-style-wrap)
+                       {:width "18%"})
+                email]
+               [ui/table-row-column
+                (merge (stylefy/use-style style-base/table-col-style-wrap)
+                       {:width "20%"})
+                [ui/icon-button {:href "#"
+                                 :on-click #(do
+                                              (.preventDefault %)
+                                              (e! (fp/->ChangePage :operator-users {:ckan-group-id ckan-group-id})))}
+                 [ic/social-people]]
```

Comment:
Is this change something that Jussi suggested?

Reply:
No, but new button describes the actions better. Moved under actions title along other action buttons, because it's an action button :)